src/model.test.ts

```diff
@@ -209,7 +412,8 @@ describe('HubApiModel', () => {
         options: {
           portal: 'https://www.arcgis.com'
         }
-      });
+      }, undefined);
```

Comment:
For this and all instances below, I'm surprised you need to include since signature of getBatchStreams has limit as an optional

Reply:
@drspacemanphd  Yes, I agree but not explicitly mentioning limit as undefined to denote absence of limit results in failed tests. However, I can add a default `limit = defined` in the function signature to avoid having to explicitly state in function in these test cases. But I am not sure if that is the right way to go about it.

Reply:
Its odd that it fails tests since an optional parameter not provided as an argument is `undefined`. A default is probably what you are going for

Reply:
@drspacemanphd Unable to fix test cases with default undefined assignment to limit parameter. Looks like Jest's `toHaveBeenNthCalledWith` requires explicitly stating undefined variable.